test/test-file.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] OPTIONAL
 You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.